examples/simpleapp/pom.xml

```diff
@@ -26,6 +26,13 @@
 			<scope>test</scope>
 		</dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on -->
```

Comment:
This is duplication. Must be removed.
Current lib is already a part of `sdk.pom` that is already present as dependency.